compute-vars/large.tfvars

```diff
@@ -0,0 +1,7 @@
+## Large Configuration Template
+
+bastion                     = {memory      = "64",   processors  = "1.5",  "count"   = 1}
+bootstrap                   = {memory      = "64",   processors  = "1.5",  "count"   = 1}
```

Comment:
Sorry a miss from me. The bootstrap should be really   {memory      = "32",   processors  = "0.5",  "count"   = 1}

Reply:
Updated these changes.